src/platform/Zephyr/wifi/WiFiManager.cpp

```diff
@@ -296,12 +296,16 @@ CHIP_ERROR WiFiManager::GetWiFiInfo(WiFiInfo & info) const
 
     if (status.state >= WIFI_STATE_ASSOCIATED)
     {
-        info.mSecurityType   = MapToMatterSecurityType(status.security);
-        info.mWiFiVersion    = MapToMatterWiFiVersionCode(status.link_mode);
-        info.mRssi           = static_cast<int8_t>(status.rssi);
-        info.mChannel        = static_cast<uint16_t>(status.channel);
-        info.mSsidLen        = status.ssid_len;
+        info.mSecurityType = MapToMatterSecurityType(status.security);
+        info.mWiFiVersion  = MapToMatterWiFiVersionCode(status.link_mode);
+        info.mRssi         = static_cast<int8_t>(status.rssi);
+        info.mChannel      = static_cast<uint16_t>(status.channel);
+        info.mSsidLen      = status.ssid_len;
+#ifdef CONFIG_WIFI_NXP
```

Comment:
Maybe we could include `zephyr/version.h` and check version:
```suggestion
#if KERNEL_VERSION_MAJOR >= 4 && KERNEL_VERSION_MINOR >= 1
```

Reply:
Thank you @adigie, update done